src/components/MovieList.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+
+import useStore from './../hooks/useStore';
+import Button from './Button';
+
+const Section = styled.section`
+	padding: 16px;
+`;
+
+const Article = styled.article`
+	max-width: 154px;
+	padding: 0;
+	animation: fadeIn 1s ${({delay}) => delay}s forwards;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+	border-radius: 20px;
+	opacity: 0;
+	background-color: #fff;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.1);
+	color: #fff;
+
+	@keyframes fadeIn {
+		0% {
+			opacity: 0;
+		}
+		100% {
+			opacity: 1;
+		}
+	}
+`;
+
+const Picture = styled.img`
+	border-radius: 20px 20px 0 0;
+`;
+
+const Grid = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	gap: 16px;
+`;
+
+const Movieheadline = styled.p`
+	padding: 8px 8px 16px 8px;
+	color: black;
+	overflow-wrap: break-word;
+`;
+
+export default function MovieList() {
+	const arrData = useStore(state => state.arrData);
+	const setData = useStore(state => state.setData);
+	const arrConfiguration = useStore(state => state.arrConfiguration);
+
+	const imagesBaseUrl = arrConfiguration.base_url + arrConfiguration.poster_sizes[1];
+
+	function fetchUrl(url) {
+		fetch(url)
+			.then(res => res.json())
+			.then(data => {
+				setData(data.results);
+			});
+	}
```

Comment:
Why did you not use the `useFetch` hook? 🤔 You do not have any error handling here. Please have a look at the `useFetch` hook as an example regarding error handling. As well, you should display a message to the user if there is a problem with the API and no data can be fetched.

Reply:
Error handling added.